packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2020,29 +2050,32 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
     return SelectionResult.none;
   }
 
-  /// Selects a word in a selectable at the location
-  /// [SelectWordSelectionEvent.globalPosition].
-  @protected
-  SelectionResult handleSelectWord(SelectWordSelectionEvent event) {
+  SelectionResult _handleSelectTextBoundary(SelectionEvent event) {
+    late final Offset effectiveGlobalPosition;
```

Comment:
Do you need this `late` here?  I assume the analyzer would have caught it if not, but just checking.

Reply:
Also maybe it's weird that effectiveGlobalPosition won't be assigned in some cases, even though it's non-nullable.  Maybe it should be `final Offset? effectiveGlobalPosition`.

Reply:
this should probably throw a unimplementedError in the else case, and the effectiveGlobalPosition can be non null